package com.bjpowernode.javase.exception;
/*
方法覆盖时遗留的问题：
    重写之后的方法不能比重写之前的异常抛出更多（更宽泛）的异常，可以更少
 */
public class ExceptionTest17 {
    public static void main(String[] args) {

    }
}
